ReactSkia/components/RSkComponent.h

```diff
@@ -40,6 +49,7 @@ struct Component {
   LayoutMetrics layoutMetrics{EmptyLayoutMetrics};
   State::Shared state{};
   int zIndex{};
+  struct props _props_;
```

Comment:
Need to change the data type and variable name.
Remove zIndex from here and move to component props